notebooks/exampledata.py

```diff
@@ -71,3 +72,26 @@ def get_example_hologram_and_background():
     get_file_from_pysilcam_blob(holo_filename)
     get_file_from_pysilcam_blob(holo_background_filename)
     return holo_filename, holo_background_filename
+
+
+def get_folder_from_holo_repository(foldername="holo_test_data_01"):
+    '''Downloads a specified folder from the holo testing repository into the working dir. if it doesn't already exist
+
+    only works for known folders that are on the GoogleDrive repository
+    by default will
+
+    Parameters
+    ----------
+    foldername : string
+        known filename on the blob
+
+    '''
+    if foldername == "holo_test_data_01":
```

Comment:
I'm not sure i undertand the if else bit here. Do both conditions do the same thing?

Reply:
Could this function be:

`def get_folder_from_holo_repository():
    url = 'https://drive.google.com/drive/folders/1yNatOaKdWwYQp-5WVEDItoibr-k0lGsP?usp=share_link'
    gdown.download_folder(url, quiet=True, use_cookies=False)
    foldername = "holo_test_data_01"
    return foldername
`

or am i missing something?

Reply:
Thanks! It is more complex than it currently needs to be - the intention was that from the Google repository it could pull one out of (in future) many different folders of holograms - but would fall back to a sensible generic option if an unknown one was requested...

Reply:
I've now completed the doc string to explain